components/resources/jobExplorer/jobList/JobListComponent.tsx

```diff
@@ -1,47 +1,63 @@
 import Image from "next/legacy/image";
 import React from "react";
+import { useSpring, animated } from "@react-spring/web";
+
 
 const disabledJobs = [];
 
-const JobGroup = ({ subheading, jobs }) => (
-  <div className="mb-12">
-    <h2 className="mb-4 text-2xl font-bold">{subheading}</h2>
-    <div className="grid grid-cols-1 gap-8 sm:grid-cols-2 md:grid-cols-3">
-      {jobs.map((job) => {
-        const isDisabled = disabledJobs.includes(job.title); // Check if job.title exists in disabledJobs array
+const JobGroup = ({ subheading, jobs }) => {
+  return (
+    <div className="mb-12">
+      <h2 className="mb-4 text-2xl font-bold">{subheading}</h2>
+      <div className="grid grid-cols-1 gap-8 sm:grid-cols-2 md:grid-cols-3">
+        {jobs.map((job) => {
+          const isDisabled = disabledJobs.includes(job.title);
+
+          // Apply individual animation to each card
+          const styles = useSpring({
```

Comment:
nice work using useSpring. Hopefully you feel comfortable enough with these animations to use in your personal projects.